toxic/provision.sh

```diff
@@ -46,8 +46,18 @@ apt-get -y install python3.8
 apt-get -y install python3-pip
 pip3 install tox
 
+# Node.js (13), npm (6.14) and JS dependencies
```

Comment:
Hmmmmmmmmm. This is a Python-specific container. I think we might want to create a new container to keep this one from becoming bloated. Let's setup a new container that is something like "nodethon - a build environment for node js applications with python"

Reply:
Ok give me a sec and I do that.